newm-server/src/main/kotlin/io/newm/server/features/earnings/database/EarningsTable.kt

```diff
@@ -0,0 +1,31 @@
+package io.newm.server.features.earnings.database
+
+import io.newm.server.features.song.database.SongTable
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.dao.id.UUIDTable
+import org.jetbrains.exposed.sql.Column
+import org.jetbrains.exposed.sql.ReferenceOption
+import org.jetbrains.exposed.sql.javatime.CurrentDateTime
+import org.jetbrains.exposed.sql.javatime.datetime
+import java.time.LocalDateTime
+import java.util.UUID
+
+object EarningsTable : UUIDTable(name = "earnings") {
+    val songId: Column<EntityID<UUID>?> =
+        reference("song_id", SongTable, onDelete = ReferenceOption.RESTRICT).nullable()
+    val stakeAddress: Column<String> = text("stake_address")
+    val amount: Column<Long> = long("amount")
+    val memo: Column<String> = text("memo")
+    val startDate: Column<Long?> = long("start_date").nullable()
+    val endDate: Column<Long?> = long("end_date").nullable()
```

Comment:
We could use `LocalDate` for startDate and endData (like we do for some date fields at SongTable).

Reply:
Changed to use LocalDateTime directly.